test/testLib.mjs

```diff
@@ -141,12 +143,21 @@ async function testResolverQueriesResults(resolverFile, queriesReferenceFolder,
             if (JSON.stringify(data, null, 2) != JSON.stringify(query.result, null, 2))
                 console.log(JSON.stringify(data, null, 2));
             
-            test(`Resolver Neptune result, ${queryFile}: ${query.name}`, async () => {    
-                expect(JSON.stringify(data, null, 2)).toBe(JSON.stringify(query.result, null, 2));
+            test(`Resolver Neptune result, ${queryFile}: ${query.name}`, async () => {
+                if(typeof query.result === 'number') { // if number
+                    expect(JSON.stringify(data, null, 2)).toBe(JSON.stringify(query.result, null, 2));
+                }
+                else if (Object.keys(query.result).length === 1 && Array.isArray(Object.values(query.result)[0])) { // if ONLY single array of objects
+                    expect(data.airportRoutesOut).toEqual(expect.arrayContaining(query.result.airportRoutesOut));
```

Comment:
Instead of hardcoding `airportRoutesOut` can you instead use `Object.keys(result)[0]` since at this point you know there is only one key and it is an array?